ngraph/core/src/pass/constant_folding.cpp

```diff
@@ -16,31 +16,36 @@
 
 #include "constant_folding.hpp"
 #include <ngraph/rt_info.hpp>
+#include "ngraph/op/util/sub_graph_base.hpp"
 
 using namespace std;
 using namespace ngraph;
 
-NGRAPH_RTTI_DEFINITION(ngraph::pass::ConstantFolding, "ConstantFolding", 0);
-
-ngraph::pass::ConstantFolding::ConstantFolding(const ngraph::BuildNodeExecutorMap& cfmap)
-    : GraphRewrite()
-    , m_cfmap{cfmap}
+bool ngraph::pass::ConstantFolding::run_on_function(std::shared_ptr<ngraph::Function> graph)
```

Comment:
```suggestion
bool ngraph::pass::ConstantFolding::run_on_function(std::shared_ptr<ngraph::Function> f)
```
Just out agreement :)